packages/devextreme/js/ui/tabs.js

```diff
@@ -696,6 +718,9 @@ const Tabs = CollectionWidget.inherit({
         switch(args.name) {
             case 'useInkRipple':
```

Comment:
we shouldn't call `this._toggleScrollingEnabledClass(args.value);` after this option change